Gems/ScriptCanvas/Assets/TranslationAssets/Nodes/Math_StringToNumber.names

```diff
@@ -27,6 +27,12 @@
                     "details": {
                         "name": "Result"
                     }
+                },
+                {
+                    "base": "DataInput_StringValue_0",
+                    "details": {
+                        "name": "StringValue"
```

Comment:
Please add a space (i.e. "String Value") to be consistent with other parameter names